src/Auth/AuthContainer.js

```diff
@@ -0,0 +1,29 @@
+import React, { PropTypes } from 'react';
+// import { bindActionCreators } from 'redux';
```

Comment:
Sure, you can keep it. But it's generally a good idea to minimize each commit and changes set for each issue.